### ISSUE TITLE
generatePDF/generateHTML: renders picture with "Syntax Error?"

### ISSUE THREAD

**Issue by mh182** [docToolchain/docToolchain#1136]
On testing #395 I noticed that the new asciidoctor doesn't render PlantUML anymore.

**To Reproduce**
1. Download end extract [example-exportStructurizr.tar.gz](https://github.com/docToolchain/docToolchain/files/11357138/example-exportStructurizr.tar.gz)
2.  Go into the directory, update docToolchain to the latest revision
```
➜ DTC_VERSION=latest ./dtcw install doctoolchain
```
3.  generate the HTML, respective PDF
```
➜ DTC_VERSION=latest ./dtcw generateHTML
dtcw 0.50 - ##DTCW_GIT_HASH##
docToolchain latest
OS/arch: Linux/x86_64
Available docToolchain environments: local sdk docker
Environments with docToolchain [latest]: local docker
Using environment: local
Using Java 11.0.18 [/home/max/.local/share/sdkman/candidates/java/11.0.18-tem/bin/java]
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.5.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build 

BUILD SUCCESSFUL in 9s
1 actionable task: 1 up-to-date
```
4. Check the generated HTML page, the diagrams are not rendered correctl
```
➜ xdg-open build/html5/example/example.html 
```

**Expected behavior**
The images generated with `exportStructurizr` are rendered correctly. The PDF output has the same result.

**Screenshots**
![image](https://user-images.githubusercontent.com/9980864/235250252-74990572-a192-4c01-8852-b99fee52f176.png)

**Configuration**
 - docToolchain version: commit 07b0a8a7
 - OS: Linux Debian unstable
 - Browser - Firefox 112.0.1
 - Relevant SDK Versions: tested with Java 11 and Java 17, same result

**Additional context**
If we call `generateHTML` with docToolchain 2.2.1, the picture is rendered correctly.

```
# Use docToolchain 2.2.1
➜ ./dtcw install doctoolchain
```

**Comment by PacoVK** [comment]
Thanks for the reproducer. What i recognized is:
* i could not run the reproducer successfully without modifications
  * directory images not found (expected at `/src/docs/` level)
  * directory structurizr not found (expected at `/src/docs/` level)
After making the changes, i could verify there is a weird behaviour. Some .puml diagrams are converted successfully (added other samples), some could not. 

Adding the same diagrams from your example to the "generateHtmlSpec" could be converted successfully 

TL;DR; needs further investigation

**Comment by PacoVK** [comment]
For the wrong directory issues, the cause is [here](https://asciidoctor.github.io/asciidoctor-gradle-plugin/development-3.x/user-guide/#_include_directives_and_base_directory)

The PlantUML issue is due to dependency conflicts, the actual converter uses `PlantUML version 1.2018.13` this is very outdated and may not be compatible with the syntax you used. Investigating furhter

**Comment by mh182** [comment]
That's strange. I do not get an error about failed includes.

Were you using `DTC_VERSION=latest` at commit 07b0a8a7? 

This reminds me, we should output the git hash if we use `latest` or `latestdev` version for easier tracking of bugs.

**Comment by PacoVK** [comment]
Okay after trolling myself for hours, i finally fixed this whole issue. We need to update `org.asciidoctor.asciidoctorj-diagram` to latest and as i expected it worked `07b0a8a` referenced a pretty old version. I will create the patch later, after i banged my head dew times against the wall because my `latest` dtc points to an old version and locally i have setup `dev`. Forget about the include errors i had before :D jsut developer things

**Comment by mh182** [comment]
Don't worry. This happened to me too. 

I was writing a reproducible bug report and when I was on the verge to create it, I noticed I was testing against 2.2.1 instead the current `ng` branch.

**Comment by mh182** [comment]
Fix verified.
